wallet/res/layout/transaction_result_content.xml

```diff
@@ -44,420 +31,397 @@
+                android:layout_marginRight="4dp"
+                android:text="@string/transaction_result_sent_to"
+                android:textSize="13sp"
+                app:layout_constraintEnd_toStartOf="@id/fee_dash_icon"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="-0.00000239 Dash" />
 
             <ImageView
-                android:id="@+id/go_to_explorer_icon"
+                android:id="@+id/fee_dash_icon"
+                android:layout_width="15dp"
+                android:layout_height="12dp"
+                app:layout_constraintBottom_toBottomOf="@+id/network_fee_label"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/network_fee_label"
+                app:srcCompat="@drawable/ic_dash_d_black"
+                app:tint="@color/content_primary" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/date_container"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingVertical="12dp">
+
+            <TextView
+                android:id="@+id/transaction_date_and_time_label"
+                style="@style/Caption.Medium.Tertiary"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_alignParentEnd="true"
-                android:src="@drawable/ic_light_gray_arrow_right"
-                />
-        </RelativeLayout>
+                android:text="@string/transaction_date_and_time"
+                android:textSize="13sp"
+                app:layout_constraintBottom_toBottomOf="@id/transaction_date_and_time"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/transaction_date_and_time" />
 
-    </androidx.cardview.widget.CardView>
+            <TextView
+                android:id="@+id/transaction_date_and_time"
+                style="@style/Caption"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:gravity="right"
+                android:maxLines="2"
+                android:text="@string/transaction_result_sent_to"
+                android:textSize="13sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="2014-01-19 01:40" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+    </LinearLayout>
 
-    <androidx.cardview.widget.CardView
-        android:id="@+id/report_issue_card"
-        app:layout_constraintTop_toBottomOf="@id/transaction_details_card"
-        style="@style/CardViewRaised"
+    <RelativeLayout
+        android:id="@+id/open_tax_category_card"
+        style="@style/ViewRaised"
```

Comment:
With the `ViewRaised` style and a background for the ripple, we can just have the view itself instead of hosting it in a `CardView`, resulting in a smaller view hierarchy footprint.

`CardView` is still useful if it's necessary to make a ripple that respects top or bottom rounded corners, like on the Explore screen.